lib/auth/clt.go

```diff
@@ -87,11 +75,8 @@ type Client struct {
 	sync.Mutex
 	ClientConfig
 	roundtrip.Client
-	transport  *http.Transport
-	conn       *grpc.ClientConn
-	grpcClient proto.AuthServiceClient
-	// closedFlag is set to indicate that the services are closed
-	closedFlag int32
+	transport *http.Transport
+	APIClient
```

Comment:
Comment explaining why it's embedded, so you don't have to write a bunch of boilerplate to satisfy the `ClientI` interface.